lib/average_end_of_day_depository_balance/main.py

```diff
@@ -126,4 +132,7 @@ async def main():
 
         print(average_end_of_day_balance)
 
+        # Dataset is set-up to return an expected value for the provided parameters
+        assert average_end_of_day_balance == 3310.5806451612902
```

Comment:
So this is true every time?

Reply:
This is true as long as the demo dataset is static. I am using this for refactors, so I know I am not breaking the results in an obvious way.

Reply:
Okay. So when we finish refactors we will remove it?

Reply:
Will bring this to parking lot. I am not sure, but I have the feeling that 99% of the team will want to remove this xD

Reply:
I will remove them to avoid adding noise to the users.